packages/odyssey-react-mui/src/ui-shell/SideNav/SideNav.tsx

```diff
@@ -142,71 +145,67 @@ const StyledSideNav = styled("nav", {
       transform: `translateX(0)`,
       transition: `opacity ${odysseyDesignTokens.TransitionDurationMain}, transform ${odysseyDesignTokens.TransitionDurationMain}`,
       width: odysseyDesignTokens.Spacing2,
-      zIndex: 2,
+      zIndex: UI_SHELL_BASE_Z_INDEX,
```

Comment:
this should be `UI_SHELL_OVERLAY_Z_INDEX`.

Reply:
Fixed!